qa/tasks/mgr/mgr_test_case.py

```diff
@@ -99,12 +99,18 @@ def setUpClass(cls):
         assert cls.mgr_cluster is not None
 
         if len(cls.mgr_cluster.mgr_ids) < cls.MGRS_REQUIRED:
-            cls.skipTest(
-                "Only have {0} manager daemons, {1} are required".format(
-                    len(cls.mgr_cluster.mgr_ids), cls.MGRS_REQUIRED))
+            cls.SKIP_TEST = True
```

Comment:
how about raising `SkipTest`. see https://docs.python.org/3/library/unittest.html#unittest.SkipTest ?

Reply:
Thanks for the review. That looks a lot better.

Reply:
Added the change, Thanks again!